modules/solid_properties/doc/content/source/userobjects/ThermalUCProperties.md

```diff
@@ -0,0 +1,47 @@
+# ThermalUCProperties
+
+!syntax description /SolidProperties/ThermalUCProperties
+
+## Description
+
+This userobject provides
+thermal properties for Uranium monocarbide as a function of temperature.
+
+!include solid_properties_units.md
+
+Isobaric specific heat is calculated from [!cite](iaea) as
+
+\begin{equation}

```

Comment:
I m not sure that s the syntax, we re getting a latex build error
maybe !equation ? what do the other pages have